.travis.yml

```diff
@@ -2,7 +2,7 @@ os: linux
 cache: pip
 dist: xenial
 language: python
-python: "3.7"
+python: "3.8"
```

Comment:
:heart_eyes:

Reply:
was necessary for protocols. also now we can use from future import annotations :)